src/tests/ftest/util/test_utils_container.py

```diff
@@ -438,7 +439,7 @@ def destroy(self, force=1):
                     # Destroy the container with the daos command
                     kwargs["pool"] = self.pool.uuid
                     kwargs["sys_name"] = self.pool.name.value
-                    kwargs["svc"] = ",".join(self.pool.svc_ranks)
+                    kwargs["svc"] = ",".join(str(self.pool.svc_ranks))
```

Comment:
This is not correct.  `join()` takes a list as an argument, therefore if you want to ensure that every item in the list is a string, you'll want to do something like this:
```suggestion
                    kwargs["svc"] = ",".join([str(item) for item in self.pool.svc_ranks])
```

Reply:
The other thing to look into is how `self.pool.svc_ranks` is being set, especially when there are multiple svc ranks.  Even though `",".join(str(x))` works (because a string is iterable, we may not get what we intend, e.g. `",".join(str(10)) = "1,0"`

Reply:
I confirmed that self.pool.svc_ranks contains the service replicas as a list of integer, so your suggested change works.